pkg/daemon/daemon.go

```diff
@@ -253,9 +264,28 @@ func (dn *Daemon) addProfileConfig(socketPath string, configFile string, nodePro
 	return nil
 }
 
+// Add fifo scheduling if specified in nodeProfile
+func addScheduling(nodeProfile *ptpv1.PtpProfile, cmdLine string) string {
+	if nodeProfile.PtpSchedulingPolicy != nil && *nodeProfile.PtpSchedulingPolicy == "SCHED_FIFO" {
+		if nodeProfile.PtpSchedulingPriority == nil {
+			glog.Errorf("Priority must be set for SCHED_FIFO; using default scheduling.")
+			return cmdLine
+		}
+		priority := *nodeProfile.PtpSchedulingPriority
+		if priority < 1 || priority > 65 {
+			glog.Errorf("Invalid priority %d; using default scheduling.", priority)
+			return cmdLine
+		}
+		return fmt.Sprintf("/bin/chrt -f %d %s", priority, cmdLine)
```

Comment:
may a log.info here to tell we change the default scheduler for process <process-name>

Reply:
okay I will make that change